MastodonSDK/Sources/MastodonUI/View/Container/MediaGridContainerView.swift

```diff
@@ -183,11 +183,12 @@ extension MediaGridContainerView {
         
         let count: Int
         let maxSize: CGSize
+		let layout: MediaLayoutResult
         
-        init(count: Int, maxSize: CGSize) {
-            self.count = min(count, 9)
+		init(count: Int, maxSize: CGSize, layout: MediaLayoutResult) {
+            self.count = min(count, 10)
             self.maxSize = maxSize
-        
+			self.layout = layout
```

Comment:
Looks like indentation got a little misaligned here? In Xcode, Editor → Structure → Re-Indent should match most indentation to the rest of the project.